index.qmd

```diff
@@ -13,19 +11,20 @@ There is no one-size-fits-all approach to cloud-optimized data, but the communit
 
 With this guide, we provide the landscape of cloud-optimized geospatial formats and provide the best-known answers to common questions.
 
-## The Opportunity
+## The opportunity
 
 Just putting data on the cloud does not solve the big geospatial data problem. Massive archives of data must be available via subsetting services in order for users to work with the data in-memory. Traditional geospatial formats are optimized for on-disk access via small internal chunks. The introduction of a network introduces  latency and the number of requests must considered. The file format must support subsetted access via adressable chunks, internal tiling or both. These characteristics allow for parallelized and partial reading.
```

Comment:
should also mention something about, Avoiding Downloading - too big to download, etc..

Reply:
still applies